scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -216,4 +216,55 @@ task ScatterIntervals {
     output {
         Array[File] scattered_interval_lists = glob("${base_filename}.scattered.*.interval_list")
     }
+}
+
+task PostprocessGermlineCNVCalls {
+    String entity_id
+    Array[File] chunk_path_tars
```

Comment:
Will this always pass chunks in the sorted order?

Reply:
Only if the output of PreprocessIntervals is sorted.  Actually, where do you check that the intervals are sorted in PostprocessGermlineCNVCalls?  I'll file an issue for us to double check.

Reply:
I meant are the files in chunk_path_tars passed according to the indices order(i.e. same order they were processed in the scatter block upstream)? The intervals ordering is checked now on the fly for each posterior record passed to the postprocessor, so it checks that next interval is in the right order compared to the previous one according to the SAMSequenceDictionary found in interval files in chunk directories

Reply:
OK, great.  I don't think we require PreprocessIntervals output to be sorted, so we should fix that.

This task is called in the workflow with `chunk_path_tars = GermlineCNVCallerCohortMode.gcnv_calls_tar`, so I think we are OK with respect to order of the scatter indices.